agent/config.go

```diff
@@ -6,7 +6,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/hashicorp/consul/lib"
+	sdkConfig "github.com/hashicorp/consul/sdk/config"
```

Comment:
Could you go with a name that is more unique without needing to name the imports? Something like how `io/ioutil` isn't `io/util` or `net/http/httptest` isn't `net/http/test`. Possibly `sdk/sdkconfig` or `sdk/configparsing`

Reply:
Certainly, which is your favorite? I think I would be leaning towards `sdk/configparsing`